testing/docker-master/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:xenial
+
+RUN apt update && apt dist-upgrade -y && apt install -y git && apt clean
+
+# enable multiverse as snapcraft cleanbuild does.
+RUN sed -i 's/ universe/ universe multiverse/' /etc/apt/sources.list
+
+# build and install snapcraft from master.
+git clone https://github.com/snapcore/snapcraft
+cd snapcraft
+apt build-dep -y ./
```

Comment:
@didrocks I tried the autoremove --purge, but it doesn't remove packages like python3-fixtures that are only build dependencies.
I added them to the remove statement. It sucks, because it can get out of date, but works for now. Pushed for a new review. Thank you!